core/app/[locale]/(default)/login/register-customer/_components/register-customer-form/index.tsx

```diff
@@ -101,20 +100,11 @@ const SubmitButton = ({ messages }: SumbitMessages) => {
   return (
     <Button
       className="relative mt-8 w-fit items-center px-8 py-2"
-      disabled={pending}
+      loading={pending}
+      loadingText={messages.submitting}
       variant="primary"
     >
-      <>
-        {pending && (
-          <>
-            <span className="absolute z-10 flex h-full w-full items-center justify-center bg-gray-400">
-              <Spinner aria-hidden="true" className="animate-spin" />
-            </span>
-            <span className="sr-only">{messages.submitting}</span>
-          </>
-        )}
-        <span aria-hidden={pending}>{messages.submit}</span>
-      </>
+      {messages.submit}
```

Comment:
Can we update the button to avoid its shifting when in loading state?

https://github.com/bigcommerce/catalyst/assets/66325265/790c45c8-841e-4143-af34-8c6e1d12d0b4

Reply:
Done